src/lib.rs

```diff
@@ -40,16 +40,27 @@ pub fn parse(contents: &str) -> Result<EditorConfigINIAST, Error<Rule>> {
 							ast.body.push(Item::Section(current_section.unwrap()));
 						}
 						current_section = Some(Section {
-							name: line.into_inner().next().unwrap().as_str().to_string(),
+							header: SectionHeader {
+								name: line.into_inner().next().unwrap().as_str().to_string(),
+								raws: Raws {
+									before: Some("".to_string()),
+									after: Some("".to_string()),
+									newline: Some("\n".to_string()),
+								},
+							},
 							body: Vec::new(),
 						});
 					}
-					Rule::property => {
+					Rule::pair => {
```

Comment:
Should we rename this from `declaration` to `pair` in the spec?

Reply:
Again - good question. Declaration is used in CSS, isn't it? I personally prefer key-value-pair but we should probably consult some additional sources as well.

Reply:
Ultimately, I want to make sure it's short, readable and no ambiguity whatsoever. That excludes `property` and `key`. Pair is pretty short and super clear. I like it.